test/functional/ui/jumpoptions_spec.lua

```diff
@@ -0,0 +1,106 @@
+-- To run this test:
+--    TEST_FILE=test/functional/example_spec.lua make functionaltest
+
+local helpers = require('test.functional.helpers')(after_each)
+local Screen = require('test.functional.ui.screen')
+local clear, feed = helpers.clear, helpers.feed
+local redir_exec = helpers.redir_exec
+local eq = helpers.eq
+
+describe('behaves like browser history when jumpoptions is history', function()
```

Comment:
```suggestion
describe('jumpoptions=history behaves like browser history', function()
```

Reply:
Fixed when moving this test into test/functional/normal/jump_spec.lua in https://github.com/neovim/neovim/pull/11530/commits/3aca2ea5bcaf9cbc1ad7eb93ca98176e753cbfd4 .